app.js

```diff
@@ -4,7 +4,16 @@ var storeLocations = [
     htmlId: 'first-and-pike',
     minCustomerPerHour: 23,
     maxCustomerPerHour: 65,
-    avgCookiesPerCustomer: 6.3
+    avgCookiesPerCustomer: 6.3,
```

Comment:
If you aren't using this file anymore, you should delete it from your repo. You'll still be able to see it if you go back through your git history, but it won't clutter up your repo.